build/x86_64_sles12/Dockerfile

```diff
@@ -28,9 +30,9 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget https://ftp.gwdg.de/pub/opensuse/repositories/home%3A/Herbster0815/openSUSE_Leap_15.1/noarch/automake-1.16.1-lp151.92.1.noarch.rpm \
- && rpm --force -Uvh automake-1.16.1-lp151.92.1.noarch.rpm \
- && rm automake-1.16.1-lp151.92.1.noarch.rpm \
+ && wget https://ftp.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Leap_15.1/noarch/automake-1.16.1-lp151.93.1.noarch.rpm \
+ && rpm --force -Uvh automake-1.16.1-lp151.93.1.noarch.rpm \
+ && rm automake-1.16.1-lp151.93.1.noarch.rpm \
```

Comment:
[Optional] We really ought to do something about having to update this all the time, e.g., change to:
```
 && zypper addrepo https://ftp.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Leap_15.1/home:Herbster0815.repo \
 && wget -O Herbster0815.key https://download.opensuse.org/repositories/home:/Herbster0815/openSUSE_Leap_15.1/repodata/repomd.xml.key \
 && rpm --import Herbster0815.key \
 && rm -f Herbster0815.key \
 && zypper -n install home_Herbster0815:automake \
```
But that's probably material for a separate PR.